### PR TITLE
doc: boards: align main board image name with board name

### PR DESCRIPTION
**Pull request by kartben**
As per board porting guidelines, the board image should be named after the board.

Compliance check failure is a false positive -- one of the images is too large, but resizing it would be counter productive at this point (old version would still live in Git history).

### CODE REVIEW

File: boards/espressif/esp_wrover_kit/doc/index.rst
Comment (sylvioalves) [inline boards/espressif/esp_wrover_kit/doc/index.rst:46]: @kartben Sounds you only renamed the image file.  Was the image re-worked as well?

**Comment by marekmatej** [comment]
LGTM

**Review by soburi — DISMISSED** [review]

File: boards/renesas/ek_ra4m2/doc/index.rst
Comment (thaoluonguw) [inline boards/renesas/ek_ra4m2/doc/index.rst:64]: @kartben : Sorry, I haven't understood the problem here.
"boards/< vendor >/<board_name>/"
Is it required to change the name of image is same with <board_name>?
Is my understanding correct?

Comment (kartben) [reply boards/renesas/ek_ra4m2/doc/index.rst:64]: Yes. It's a convention that's important as it's what allows to know, in case there are multiple images for a board, which one actually depicts the board. See https://github.com/zephyrproject-rtos/zephyr/pull/79160 and how this particular board shows up with a "wrong" image. Thanks!

**Review by thaoluonguw — DISMISSED** [review]

**Review by wmrsouza — DISMISSED** [review]

**Review by nashif — DISMISSED** [review]

**Review by uLipe — DISMISSED** [review]

**Review by sylvioalves — DISMISSED** [review]

**Comment by kartben** [comment]
hadn't noticed docs CI failure until now, sorry. Compliance check will still be failing and will need super-power merge rights.

**Review by thaoluonguw — APPROVED** [review]

**Review by uLipe — APPROVED** [review]

**Review by wmrsouza — APPROVED** [review]

**Comment by kartben** [comment]
@sylvioalves  please re-approve if you have a chance

**Review by sylvioalves — APPROVED** [review]

**Pull request merged**
